src/JustEat.StatsD/DateTimeExtensions.cs

```diff
@@ -12,9 +12,10 @@ public static class DateTimeExtensions
         /// <summary>	Returns the DateTime as the number of seconds since the epoch (1970), which is Unix time. </summary>
         /// <param name="dateTime">	The dateTime to act on. </param>
         /// <returns>	A number of seconds since the epoch. </returns>
-        public static double AsUnixTime(this DateTime dateTime)
+        public static long AsUnixTime(this in DateTime dateTime)
         {
-            return Math.Round(dateTime.ToUniversalTime().Subtract(Epoch).TotalSeconds);
+            DateTimeOffset x = dateTime;
```

Comment:
Relying on the implicit cast can cause issues with time zone conversions if `dateTime` isn't UTC.